<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCalibration</class>
 <widget class="QDialog" name="DialogCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校正</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>数据源</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>191</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>171</width>
           <height>72</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="autoGetData">
            <property name="text">
             <string>发送“获取数据”信号</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>间隔(ms):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="autoGetDataInterval">
              <property name="text">
               <string>50</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>获取的向量</string>
        </property>
        <widget class="QComboBox" name="dataType">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="title">
        <string>采集</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>0</y>
          <width>211</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QGroupBox" name="preOffset">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="title">
            <string>预偏移</string>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>191</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>x:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="preOffset_x">
               <property name="text">
                <string>0.0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="preOffset_y">
               <property name="text">
                <string>0.0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>z:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="preOffset_z">
               <property name="text">
                <string>0.0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>采集到的样本(可以手动输入)：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="sample">
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>x:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="display_x">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="display_y">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>z:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="display_z">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>10</y>
              <width>97</width>
              <height>53</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="bt_input">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>采集一批</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>采样数：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="samplesPerTime">
                 <property name="text">
                  <string>16</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="bt_clear">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
           <widget class="QPushButton" name="bt_refleshDisplay">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>刷新显示</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>计算</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>201</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>加速度计和罗盘</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>181</width>
           <height>231</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>目标球的半径:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="radius">
              <property name="text">
               <string>1.0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="bt_calculate">
            <property name="text">
             <string>计算椭球参数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>结果(已加预偏移)：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="result">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>110</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;拟合公式为：&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;x_c = (x_m + X_OFFSET) * X_GAIN; &lt;br /&gt;y_c = (y_m + Y_OFFSET) * Y_GAIN;&lt;br /&gt;z_c = (z_m + Z_OFFSET) * Z_GAIN;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btMakeAccParam">
            <property name="text">
             <string>作为加速度计校正参数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btMakeMagParam">
            <property name="text">
             <string>作为罗盘校正参数</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>270</y>
          <width>201</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>陀螺</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>181</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="btMakeGyrOffset">
            <property name="text">
             <string>计算均值，并作为陀螺偏移值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leGyrAverage">
            <property name="toolTip">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;这是&lt;span style=&quot; font-weight:600;&quot;&gt;平均值&lt;/span&gt;，即&lt;span style=&quot; font-weight:600;&quot;&gt;偏移值&lt;/span&gt;。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;偏移的校正值&lt;/span&gt;是&lt;span style=&quot; font-weight:600;&quot;&gt;偏移值&lt;/span&gt;的相反数。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>校验</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>4</y>
          <width>211</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QCheckBox" name="displayEllipsoid">
           <property name="text">
            <string>显示拟合椭球</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btCheckValue">
           <property name="text">
            <string>显示校正后的数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btCheckLength">
           <property name="text">
            <string>显示长度及误差</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textCheck"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="WidgetView" name="view" native="true">
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetView</class>
   <extends>QWidget</extends>
   <header>DialogCalibration/widgetview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>autoGetData</tabstop>
  <tabstop>dataType</tabstop>
  <tabstop>autoGetDataInterval</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>bt_input</tabstop>
  <tabstop>preOffset_x</tabstop>
  <tabstop>preOffset_y</tabstop>
  <tabstop>preOffset_z</tabstop>
  <tabstop>bt_clear</tabstop>
  <tabstop>bt_refleshDisplay</tabstop>
  <tabstop>samplesPerTime</tabstop>
  <tabstop>sample</tabstop>
  <tabstop>display_x</tabstop>
  <tabstop>display_y</tabstop>
  <tabstop>display_z</tabstop>
  <tabstop>radius</tabstop>
  <tabstop>bt_calculate</tabstop>
  <tabstop>result</tabstop>
  <tabstop>displayEllipsoid</tabstop>
  <tabstop>btCheckValue</tabstop>
  <tabstop>btCheckLength</tabstop>
  <tabstop>textCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
